cmd/process-agent/collector.go

```diff
@@ -175,11 +177,15 @@ func (l *Collector) runCheckWithRealTime(c checks.CheckWithRealTime, results, rt
 		return
 	}
 	l.messagesToResults(start, c.Name(), run.Standard, results)
-	l.messagesToResults(start, c.RealTimeName(), run.RealTime, rtResults)
-
 	if options.RunStandard {
+		exports.StoreCheckOutput(c.Name(), run.Standard)
 		logCheckDuration(c.Name(), start, runCounter)
 	}
+
+	l.messagesToResults(start, c.RealTimeName(), run.RealTime, rtResults)
+	if options.RunRealTime {
+		exports.StoreCheckOutput(c.RealTimeName(), run.RealTime)
+	}
```

Comment:
I really like this solution - we are now able to add running check details in the `status` command with this.

Reply:
That was my intention. I thought about this before when I was doing the `status` command, and thought it may be useful to have somewhere where we can export things for reference by other packages.